app/controllers/admin_controller.rb

```diff
@@ -37,16 +37,16 @@ def set_bust_cache
 
   private
 
+  # We only want to set this via query string, as it now defaults to enabled globally
   def set_reporting_schema_v2
-    RequestStore[:reporting_schema_v2] = reporting_schema_via_param_or_feature_flag
+    if params.key?(:v2)
+      param_flag = ActiveRecord::Type::Boolean.new.deserialize(params[:v2])
+      Reports.reporting_schema_v2 = param_flag
+    end
```

Comment:
This can be simplified now because we know the value will be true everywhere, but we just want the ability to override to false for a little bit to confirm any discrepancies in prod.